src/advanced/components/AdminTabs.tsx

```diff
@@ -0,0 +1,33 @@
+interface AdminTabsProps {
+  activeTab: "products" | "coupons";
+  handleActiveTab: (value: "products" | "coupons") => void;
+}
+
+const AdminTabs = ({ activeTab, handleActiveTab }: AdminTabsProps) => {
+  return (
+    <nav className="-mb-px flex space-x-8">
+      <button
+        onClick={() => handleActiveTab("products")}
+        className={`py-2 px-1 border-b-2 font-medium text-sm transition-colors ${
+          activeTab === "products"
+            ? "border-gray-900 text-gray-900"
+            : "border-transparent text-gray-500 hover:text-gray-700 hover:border-gray-300"
+        }`}
+      >
+        상품 관리
+      </button>
+      <button
+        onClick={() => handleActiveTab("coupons")}
+        className={`py-2 px-1 border-b-2 font-medium text-sm transition-colors ${
+          activeTab === "coupons"
+            ? "border-gray-900 text-gray-900"
+            : "border-transparent text-gray-500 hover:text-gray-700 hover:border-gray-300"
+        }`}
+      >
+        쿠폰 관리
+      </button>
+    </nav>
+  );
+};
```

Comment:
```suggestion
export function AdminTab({
  tabs,
  activeTab,
  setActiveTab,
}: {
  tabs: {
    label: string;
    value: "products" | "coupons";
  }[];
  activeTab: "products" | "coupons";
  setActiveTab: React.Dispatch<React.SetStateAction<"products" | "coupons">>;
}) {
  return (
    <div className="border-b border-gray-200 mb-6">
      <nav className="-mb-px flex space-x-8">
        {tabs.map((tab) => (
          <button
            key={tab.value}
            onClick={() => setActiveTab(tab.value)}
            className={`py-2 px-1 border-b-2 font-medium text-sm transition-colors ${
              activeTab === tab.value
                ? "border-gray-900 text-gray-900"
                : "border-transparent text-gray-500 hover:text-gray-700 hover:border-gray-300"
            }`}
          >
            {tab.label}
          </button>
        ))}
      </nav>
    </div>
  );
}
```

지금은 탭의 형태와 구조가 고정적인데 외부에서 변수를 변경함으로써 탭이 늘어나고 줄어들 수 있는 구조로 변경하는 것은 어떨까요?